modules/analysis.nf

```diff
@@ -39,9 +39,7 @@ process download_nextclade_files {
     """
     home=PWD
     nextclade_ver=`(nextclade -v)`
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/tree.json
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/genemap.gff
-    wget -P nextclade_files https://raw.githubusercontent.com/nextstrain/nextclade/\${nextclade_ver}/data/sars-cov-2/qc.json 
+    nextclade dataset get --name 'sars-cov-2' --output-dir nextclade_files
```

Comment:
I do not intend to change this. Should I revert back to the previous version?

Reply:
This was introduced by Nick as part of 347803c